test/ome-files/tiff.cpp

```diff
@@ -153,7 +164,7 @@ namespace
         idx[ome::files::DIM_CHANNEL] = idx[ome::files::DIM_MODULO_Z] =
         idx[ome::files::DIM_MODULO_T] = idx[ome::files::DIM_MODULO_C] = 0;
 
-      const char * const shades[] = {" ", "░", "▒", "▓", "█"};
+      const std::array<const char *,5> shades{{" ", "░", "▒", "▓", "█"}};
```

Comment:
@mtbc :+1: see https://www.openmicroscopy.org/site/support/contributing/code-contributions.html?highlight=utf#character-encoding, I think this makes sense to generalize this statement to other languages (dunno whether this applies to technologies like `ansible`)